### ISSUE TITLE
Install_dispatcher requires data volume mountpoint even when not setting this up

### ISSUE THREAD

**Issue by henrykuijpers** [shinesolutions/puppet-aem-curator#183]
Since: https://github.com/shinesolutions/puppet-aem-curator/commit/97057e2603b45961a068ab0596c358cc499cf1c8#diff-2d796cd6a9229c68056f737ef74930dd
The Dispatcher_install step now fails with the following error:

```
    dispatcher: Error: Evaluation Error: Error while evaluating a Resource Statement, Class[Aem_curator::Install_dispatcher]:
    dispatcher:   expects a value for parameter 'data_volume_device'
    dispatcher:   expects a value for parameter 'data_volume_mount_point' (file: /vagrant/aempuppet/modules/aemprovisioning/manifests/dispatcher/installation.pp, line: 13, column: 7) on node dispatcher.localhost.com
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/types/type_mismatch_describer.rb:539:in `validate_parameters'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/types/type_mismatch_describer.rb:499:in `validate_parameters'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/resource/type.rb:381:in `validate_resource_hash'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/resource/type.rb:339:in `set_resource_parameters'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/resource/type.rb:123:in `evaluate_code'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/resource.rb:79:in `block in evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/resource.rb:71:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:386:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:386:in `evaluate_classes'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_resource_support.rb:61:in `block in create_resources'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_resource_support.rb:37:in `map'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_resource_support.rb:37:in `create_resources'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_support.rb:341:in `create_resources'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:865:in `block in eval_ResourceExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:862:in `map'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:862:in `eval_ResourceExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:90:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `block in eval_BlockExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `reduce'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `eval_BlockExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:90:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:177:in `block in evaluate_block_with_bindings'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/scope.rb:983:in `with_guarded_scope'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:173:in `evaluate_block_with_bindings'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/closure.rb:233:in `block in call_with_scope'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/closure.rb:232:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/closure.rb:232:in `call_with_scope'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/closure.rb:188:in `block in call'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/closure.rb:187:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/closure.rb:187:in `call'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:973:in `block in proc_from_lambda'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/functions/each.rb:145:in `block in foreach_Enumerable_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/functions/each.rb:145:in `loop'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/functions/each.rb:145:in `foreach_Enumerable_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatch.rb:60:in `invoke'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatcher.rb:43:in `block in dispatch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatcher.rb:42:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatcher.rb:42:in `dispatch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/function.rb:46:in `block in call'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/function.rb:45:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/function.rb:45:in `call'
    dispatcher: /vagrant/aempuppet/modules/aemprovisioning/manifests/dispatcher/installation.pp:6
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/puppet_stack.rb:39:in `stack'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_support.rb:305:in `block in call_function'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_support.rb:303:in `call_function'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:966:in `call_function_with_block'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:959:in `eval_CallMethodExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:90:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:1028:in `block in eval_IfExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/scope.rb:983:in `with_guarded_scope'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:1026:in `eval_IfExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:48:in `block in visit_this'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:42:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:42:in `visit_this'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:92:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `block in eval_BlockExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `reduce'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:660:in `eval_BlockExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:90:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/parser/evaluating_parser.rb:63:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:32:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:72:in `block (2 levels) in evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:71:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:71:in `block in evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:70:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:70:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast.rb:31:in `safeevaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/resource/type.rb:136:in `evaluate_code'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/resource.rb:79:in `block in evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/resource.rb:71:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:386:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:386:in `evaluate_classes'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/functions/include.rb:48:in `include'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatch.rb:60:in `invoke'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatcher.rb:43:in `block in dispatch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatcher.rb:42:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/dispatcher.rb:42:in `dispatch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/function.rb:46:in `block in call'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/function.rb:45:in `catch'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/functions/function.rb:45:in `call'
    dispatcher: /vagrant/sligro/manifests/installation.pp:3
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/puppet_stack.rb:41:in `stack'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_support.rb:305:in `block in call_function'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/runtime3_support.rb:303:in `call_function'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:964:in `call_function_with_block'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:959:in `eval_CallMethodExpression'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:48:in `block in visit_this'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:42:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:42:in `visit_this'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:92:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /vagrant/sligro/manifests/installation.pp:0
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/puppet_stack.rb:41:in `stack'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:737:in `eval_Program'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:48:in `block in visit_this'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:42:in `each'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:42:in `visit_this'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/visitor.rb:92:in `visit_this_1'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/evaluator/evaluator_impl.rb:81:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/pops/parser/evaluating_parser.rb:63:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast/pops_bridge.rb:125:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/ast.rb:31:in `safeevaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/resource/type.rb:136:in `evaluate_code'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/resource.rb:79:in `block in evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/resource.rb:71:in `evaluate'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:592:in `evaluate_main'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:166:in `block (2 levels) in compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:166:in `block in compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/context.rb:65:in `override'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet.rb:260:in `override'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:155:in `compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/parser/compiler.rb:35:in `compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/indirector/catalog/compiler.rb:303:in `block (2 levels) in compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler/around_profiler.rb:58:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/profiler.rb:51:in `profile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/indirector/catalog/compiler.rb:301:in `block in compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util.rb:232:in `block in benchmark'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util.rb:231:in `benchmark'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/indirector/catalog/compiler.rb:299:in `compile'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/indirector/catalog/compiler.rb:54:in `block in find'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/node/environment.rb:450:in `with_text_domain'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/indirector/catalog/compiler.rb:53:in `find'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/indirector/indirection.rb:198:in `find'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application/apply.rb:262:in `block in main'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/context.rb:65:in `override'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet.rb:260:in `override'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application/apply.rb:233:in `main'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application/apply.rb:174:in `run_command'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application.rb:375:in `block in run'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util.rb:690:in `exit_on_fail'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application.rb:375:in `run'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/command_line.rb:139:in `run'
    dispatcher: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/command_line.rb:77:in `execute'
    dispatcher: /opt/puppetlabs/puppet/bin/puppet:5:in `<main>'
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by mbloch1986** [comment]
@henrykuijpers Thanks for informing us!

I do have a fix for it :

https://github.com/mbloch1986/puppet-aem-curator/tree/fix-183/fix_install_dispatcher

I am planning to test in the next few days. I'll let you know how the testing goes once done.

Cheers

**Comment by mbloch1986** [comment]
@henrykuijpers changes are included in relese 3.11.0. Please re-open the ticket if you are still having troubles! Thanks
